docs/ref/contrib/auth.txt

```diff
@@ -182,6 +182,11 @@ Methods
         You may need this if authentication for your application takes place
         against an existing external source such as an LDAP directory.
 
+        .. note::
+            Users marked with an unusable password will not
+            able to request a password reset email via
+            :class:`~django.contrib.auth.views.PasswordResetView`.
```

Comment:
We try to avoid having bare notes in the docs, we prefer admonitions with meaningful titles. For example:

```suggestion
        .. admonition:: Password reset restriction
            Users having an unusable password will not able to request a
            password reset email via
            :class:`~django.contrib.auth.views.PasswordResetView`.
```

Reply:
Thank you! I've incorporated your suggestion in my patch :)